src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -294,7 +354,7 @@ public IIntellisenseResult Suggest(CheckResult checkResult, int cursorPosition)
         [Obsolete("Use overload without expression")]
         public IIntellisenseResult Suggest(string expression, CheckResult checkResult, int cursorPosition)
```

Comment:
>expression [](http://example.com/codeflow?start=50&length=10)

test (null + length) + checkResult + cursorPosition in range

Reply:
existing code. out of scope.